dex/testing/polygon/harness.sh

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env bash
+
+SESSION="polygon-harness"
+
+SOURCE_DIR=$(pwd)
+NODES_ROOT=~/dextest/polygon
+GENESIS_JSON_FILE_LOCATION="${NODES_ROOT}/genesis.json"
+HARNESS_DIR=${NODES_ROOT}/harness-ctl
+
+mkdir -p "${NODES_ROOT}/${NAME}"
+mkdir -p "${HARNESS_DIR}"
+
+NAME="alpha"
+PASSWORD="abc"
+
+cat > "${HARNESS_DIR}/${NAME}" <<EOF
+#!/usr/bin/env bash
+bor \$*
```

Comment:
Is it possible to do something like `bor "$1" --datadir=${NODE_DIR} "${@: 2}"`?

Reply:
Each `bor` command accepts limited args. Most `bor` commands do not require the `--datadir` flag except for the `server` cmd. I'd like to remove the script entirely unless there is a use case for it.

Reply:
Well, `attach` clearly requires it, otherwise it doesn't know where the bor.ipc file is.

Reply:
> Well, `attach` clearly requires it, otherwise it doesn't know where the bor.ipc file is.

Yeah, `attach` requires the path to the `bor.ipc` file which exists at `datadir` which is always `datadir/bor.ipc` unless it was changed via the `bor server  --ipcpath another/path/within/datadir`

Reply:
Removed this script in https://github.com/decred/dcrdex/pull/2399/commits/34110e3a8356499a40423778a0a882ab58d71352